packages/next/src/build/webpack/plugins/telemetry-plugin/telemetry-plugin.ts

```diff
@@ -127,16 +119,17 @@ const useCacheTracker = createUseCacheTracker()
 /**
  * Determine if there is a feature of interest in the specified 'module'.
  */
-function findFeatureInModule(module: Module): Feature | undefined {
+function findFeatureInModule(module: webpack.Module): Feature | undefined {
   if (module.type !== 'javascript/auto') {
     return
   }
-  const normalizedIdentifier = module.identifier().replace(/\\/g, '/')
+
   for (const [feature, path] of FEATURE_MODULE_MAP) {
-    if (normalizedIdentifier.endsWith(path)) {
+    if ((module as webpack.NormalModule).resource.endsWith(path)) {
```

Comment:
`module.identifier()` includes the layer e.g. `next/image.js|some-layer` so the old matching approach didn't work. `resource` does not contain just the path.

Reply:
I don't see `resourcePath` on the module. There's `resourceResolveData?.path?` but this is untyped so I don't trust it. Probably best to wait for a concrete case.

Was the resource query not part of the module identifier?

Reply:
oh sorry I got it wrong with the loader API, module.resource should be enough